microsite/data/plugins/ibm-apic-backend.yaml

```
@@ -0,0 +1,16 @@
+---
+title: IBM APIC Backend
```

```suggestion
title: IBM APIC EntityProvider
```

Would that make more sense? Seems more clear to me as an outsider to IBM APIC

---

I agree with you. Will update the name.